### ISSUE TITLE
Disable persona creation wizard/redirect to static theme wizard

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#7846]
When we launch static themes we need to redirect the old LWT wizard url to the new page and change any appropriate links/menu items.

Open questions:
- do we need all the current entry points - e.g. should there be a create a theme button when we have the wizard button inside the add-on flow.  If so where should it go - the wizard directly or do the same as the add-on flow?
- should we allow existing LWTs to be edited in the period between static themes launching and the existing LWT being migrated to a static theme?

@jvillalobos ^

**Comment by eviljeff** [comment]
Note: we may want a separate waffle for this so QA can still test the old persona wizard without disabling all static theme uploads on -dev.

**Comment by jvillalobos** [comment]
> do we need all the current entry points - e.g. should there be a create a theme button when we have the wizard button inside the add-on flow. If so where should it go - the wizard directly or do the same as the add-on flow?

Further down the line I expect we will have a "Create a New Extension" button and a "Create a New Theme" button, both leading to the same point, since the flow is the same. So, let's keep the existing "Create a New Theme" access points, but have them point to the beginning of the submission flow, not directly to the wizard. Once this is done on -dev, @caitmuenster and I will create some kind of guide for existing theme designers, so they aren't confused.

> should we allow existing LWTs to be edited in the period between static themes launching and the existing LWT being migrated to a static theme?

That would be ideal.

**Comment by eviljeff** [comment]
QA:
to enable this you need to add and enable a waffle switch named `disable-lwt-uploads` in /admin
Once enabled all the links to the old LWT wizard should point to the add submission flow instead, e.g.:
- the tools menu in devhub/legacy frontend when you're logged in 
- the legacy frontend browse pages (just Theme, afaik)
- devhub links to submit a theme on the page

In addition, the url to the old LWT submission page should redirect to the add-on submission flow (https://addons.mozilla.org/developers/theme/submit)

Existing LWT themes should still be editable.

**Comment by ioanarusiczki** [comment]
I added and then enabled the waffle switch disable-lwt-uploads in /admin.
I checked that Submit a new theme redirects to the add-on submission flow from:
- devhub - tools - submit a new theme
- devhub - manage my submissions page - submit a new theme
- devhub https://addons-dev.allizom.org/en-US/developers/ submit a new theme button
- new frontend AMO homepage and the legacy frontend
 The existing LWT are still editable: upload new design, editing name or the URL of the detail page.

I disabled the waffle - the lightweight theme actual submission page will be displayed again https://addons-dev.allizom.org/en-US/developers/theme/submit

Verified on AMO dev with FF59 (Win10).

![enabling waffle switch](https://user-images.githubusercontent.com/33448286/38254098-0c87d758-3761-11e8-91fe-22235a921df0.gif)
